src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests/Controllers/DataSetsControllerTests.cs

```diff
@@ -703,4 +707,479 @@ private async Task<HttpResponseMessage> GetVersion(Guid dataSetId, string dataSe
+                        case LocationLocalAuthorityOptionMeta locationLocalAuthorityOptionMeta:
+                            var locationLocalAuthorityOptionMetaViewModel = 
+                                Assert.IsType<LocationLocalAuthorityOptionMetaViewModel>(locationOptionMetaViewModel);
+                            Assert.Equal(
+                                locationLocalAuthorityOptionMeta.Label, 
+                                locationLocalAuthorityOptionMetaViewModel.Label);
+                            Assert.Equal(
+                                locationLocalAuthorityOptionMeta.Code, 
+                                locationLocalAuthorityOptionMetaViewModel.Code);
+                            Assert.Equal(
+                                locationLocalAuthorityOptionMeta.OldCode, 
+                                locationLocalAuthorityOptionMetaViewModel.OldCode);
+                            break;
+                        case LocationProviderOptionMeta locationProviderOptionMeta:
+                            var locationProviderOptionMetaViewModel = 
+                                Assert.IsType<LocationProviderOptionMetaViewModel>(locationOptionMetaViewModel);
+                            Assert.Equal(
+                                locationProviderOptionMeta.Label, 
+                                locationProviderOptionMetaViewModel.Label);
+                            Assert.Equal(
+                                locationProviderOptionMeta.Ukprn, 
+                                locationProviderOptionMetaViewModel.Ukprn);
+                            break;
+                        case LocationRscRegionOptionMeta locationRscRegionOptionMeta:
+                            var locationRscRegionOptionMetaViewModel = 
+                                Assert.IsType<LocationRscRegionOptionMetaViewModel>(locationOptionMetaViewModel);
+                            Assert.Equal(
+                                locationRscRegionOptionMeta.Label, 
+                                locationRscRegionOptionMetaViewModel.Label);
+                            break;
+                        case LocationSchoolOptionMeta locationSchoolOptionMeta:
+                            var locationSchoolOptionMetaViewModel = 
+                                Assert.IsType<LocationSchoolOptionMetaViewModel>(locationOptionMetaViewModel);
+                            Assert.Equal(
+                                locationSchoolOptionMeta.Label, 
+                                locationSchoolOptionMetaViewModel.Label);
+                            Assert.Equal(
+                                locationSchoolOptionMeta.Urn, 
+                                locationSchoolOptionMetaViewModel.Urn);
+                            Assert.Equal(
+                                locationSchoolOptionMeta.LaEstab, 
+                                locationSchoolOptionMetaViewModel.LaEstab);
+                            break;
```

Comment:
Bit of a nitpick, but all of this is a bit noisy and on the hard to read side. Maybe we could simplify the variables here by dropping the 'location' parts e.g.

```cs
case LocationCodedOptionMeta codedMeta:
    var codedViewModel =
        Assert.IsType<LocationCodedOptionMetaViewModel>(locationOptionMetaViewModel);
    Assert.Equal(codedMeta.Label, codedViewModel.Label);
    Assert.Equal(codedMeta.Code, codedViewModel.Code);
```

Reply:
I'd agree with that!

also now allows me to put stuff on 1 line without hitting line limits :D